authors/apps/authentication/serializers.py

```diff
@@ -137,44 +133,31 @@ class UserSerializer(serializers.ModelSerializer):
         min_length=8,
         write_only=True
     )
+    profile = ProfileSerializer()
 
     class Meta:
         model = User
-        fields = ('email', 'username', 'password', 'token',)
-
-        # The `read_only_fields` option is an alternative for explicitly
-        # specifying the field with `read_only=True` like we did for password
-        # above. The reason we want to use `read_only_fields` here is because
-        # we don't need to specify anything else about the field. For the
-        # password field, we needed to specify the `min_length` and
-        # `max_length` properties too, but that isn't the case for the token
-        # field.
-
```

Comment:
I thought they were redundant because there are docstrings already in place. For the purposes of maintenance and tracking, I have rewritten the comments.